spec/controllers/collections_controller_spec.rb

```diff
@@ -9,4 +9,14 @@
       expect(assigns[:page_subtitle]).to eq("#{collection.title} - Subcollections")
     end
   end
+
+  describe "GET #load_collection_from_id" do
+    context "when id does not exist" do
```

Comment:
It'd be better to make this about the action, since that is what's getting tested:
```suggestion
  describe "GET #show" do
    context "when collection does not exist" do
```